filament/backend/include/backend/Platform.h

```diff
@@ -81,9 +81,9 @@ class UTILS_PUBLIC DefaultPlatform : public Platform {
      *                      accordingly. Can't be nullptr.
      *
      * @param nativeDisplay Specify a display to connect to. Currently only meaningful for the EGL
-     *                      platform, on which you should pass in an initialized EGLDisplay 
+     *                      platform, on which you should pass in an initialized EGLDisplay
```

Comment:
while here: reading 'initialized `EGLDisplay`' suggested to me that `eglInitialize(nativeDisplay)` must be called before passing it to Filament. I know `eglInitialize` is for initializing a _display connection_ and not a _display_ and it might just be my fault. In case you agree, the comment could be:
```suggestion
     *                      platform, on which you should pass in an EGLDisplay
```